pinot-common/src/main/java/org/apache/pinot/common/restlet/resources/SegmentStatus.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.common.restlet.resources;
+
+/**
+ * Holds segment last reload time status along with any errors for a segment with unsuccessful call to get reload times.
+ */
+public class SegmentStatus {
+  public String _segmentName;
+  public String _segmentReloadTime;
```

Comment:
* Why is this a "String"? Why not 'long'?
* Please clearly indicate the time unit in the field name. e.g. `segmentReloadTimeUTCMs`
* What is the value in this field if the segment was never reloaded, or the server restarts after the reload? Or, the reload faced an error and failed?